src/Console/InfectionContainer.php

```diff
@@ -219,7 +220,7 @@ public static function create(): self
                 return new VersionParser();
             },
             Lexer::class => static function (): Lexer {
-                return new Lexer\Emulative([
+                return new Emulative([
```

Comment:
@ TomasVotruba is it expected that it does this change although it was not requested?

Reply:
See https://github.com/rectorphp/rector#import-use-statements